docs/logging-install.md

```diff
@@ -0,0 +1,148 @@
+openshift_logging_elasticsearch_nodeselector={'node-type': 'logging'}
+
+# logging stack for ops
+openshift_logging_use_ops=true
+openshift_logging_es_ops_cluster_size=1
+openshift_logging_es_ops_memory_limit=2G
+
+# uncomment the following to enable htpasswd authentication; defaults to DenyAllPasswordIdentityProvider
+openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+
+# default selectors for router and registry services
+openshift_router_selector='region=infra'
+openshift_registry_selector='region=infra'
+openshift_disable_check=docker_storage,memory_availability
+
+# host group for masters
+[masters]
+os-master-[1:2].lon1.centos.org
+
+# host group for etcd
+[etcd]
+os-node-[1:2].lon1.centos.org
+
+# host group for nodes, includes region info
+[nodes]
+os-master-1.lon1.centos.org openshift_node_labels="{'region': 'infra','zone': 'default','purpose':'infra', 'node-type': 'logging'}" openshift_schedulable=true openshift_ip=172.29.33.36
+os-master-2.lon1.centos.org openshift_node_labels="{'region': 'infra','zone': 'default','purpose':'infra', 'node-type': 'logging'}" openshift_schedulable=true openshift_ip=172.29.33.46
+os-node-1.lon1.centos.org openshift_node_labels="{'region':'primary','zone': 'default','purpose':'prod', 'node-type': 'logging'}" openshift_schedulable=true openshift_ip=172.29.33.23
+os-node-2.lon1.centos.org openshift_node_labels="{'region':'primary','zone': 'default','purpose':'prod', 'node-type': 'logging'}" openshift_schedulable=true openshift_ip=172.29.33.52
+```
+
+We are setting up two logging stacks. Both the stacks are based on EFK
+(Elasticsearch - Fluentd - Kibana). First stack is defined by:
+
+```
+# logging stack
+openshift_logging_install_logging=true
+openshift_logging_es_cluster_size=1
+openshift_logging_es_memory_limit=2G
+openshift_logging_elasticsearch_storage_type="hostmount"
+openshift_logging_elasticsearch_hostmount_path="/logging"
+openshift_logging_elasticsearch_nodeselector={'node-type': 'logging'}
+```
+
+This is going to use the local host directory on a node which is labeled with
+`'node-type': 'logging'` which in case is all the nodes in the cluster. What
+this does is spin up EFK stack and use `/logging` directory from one of the
+nodes satisfying the mentioned label. 
+
+```
+# logging stack for ops
+openshift_logging_use_ops=true
+openshift_logging_es_ops_cluster_size=1
+openshift_logging_es_ops_memory_limit=2G
+```
+
+This is again going to use `/logging` directory in one of the nodes with
+correct label. It's going to spin up pods for Elasticsearch and Kibana while
+single Fluentd pod in the logging namespace is going to take care of sending
+logs to the relevant cluster.
+
+At the moment, we are using worst possible Linux file persmissions so that
+Elasticsearch pods can access the `/logging` directory:
+
+```bash
+$ chmod 777 /logging
+```
```

Comment:
I think we need a step to create `/logging` directory?

Reply:
I didn't mention that because earlier in the document, we've mentioned that `/logging` directory is going to be used for storing the logs.